From 98365ce5ecf835841344bf1b6ec34f00223a4ca5 Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Thu, 17 Jun 2021 08:05:25 +0000
Subject: [PATCH] src/Makefile.am: Add read_write_all.c to common_SOURCES

It is required by image.c.

Fixed:
src/image.c:659: undefined reference to `write_all'

Upstream-Status: Pending

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 src/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index 38f93ff..b32befc 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -9,7 +9,7 @@ AM_CFLAGS += -DOPENSSL_API_COMPAT=0x10100000L
 endif
 
 common_SOURCES = idc.c idc.h image.c image.h fileio.c fileio.h \
-	efivars.h $(coff_headers)
+	../lib/ccan.git/ccan/read_write_all/read_write_all.c efivars.h $(coff_headers)
 common_LDADD = ../lib/ccan/libccan.a $(libcrypto_LIBS)
 common_CFLAGS = -I$(top_srcdir)/lib/ccan/ -Werror
 
-- 
2.25.1

